Libs/RaptureCore/src/main/java/rapture/kernel/StructuredApiImpl.java

```diff
@@ -185,7 +185,11 @@ public void dropTable(CallingContext context, String tableUri) {
     @Override
     public Boolean tableExists(CallingContext context, String tableUri) {
         RaptureURI uri = new RaptureURI(tableUri, Scheme.STRUCTURED);
-        return getRepoOrFail(uri.getAuthority()).tableExists(uri.getDocPath());
+        try {
+            return getRepoOrFail(uri.getAuthority()).tableExists(uri.getDocPath());
+        } catch (Exception e) {
```

Comment:
RaptureException is a subclass of Exception. It'll catch anything except Errors or other Throwables.